docs/iris/src/userguide/real_and_lazy_data.rst

```diff
@@ -71,27 +68,27 @@ remain lazy::
     >>> cube.has_lazy_data()
     True
 
-This is referred to as **lazy evaluation**.
+The process by which the operation is deferred until the result is requested is
+referred to as **lazy evaluation**.
 
 Certain operations, including regridding and plotting, can only be run on real data.
 Calling such operations on lazy data will automatically realise your lazy data.
 
 You can also realise (and so load into memory) your cube's lazy if you 'touch' the data.
```

Comment:
I'm still not sure about 'your cube's lazy'.  I don't know whether to interpret it as 'your cube is lazy' or 'the lazy of your cube'.  Logically, the latter seems more befitting of the sentence, but then you are using lazy as a noun.  I know that I am only nit-picking here (is that the right type of nit for that expression?), but something about that turn of phrase just doesn't sit right with me.  It is not a technical issue, so if you really want to keep it as it is then I will still merge it, but as it's you Pete I thought you would appreciate a small debate about grammar.